src/FortuneWindow.cpp

```diff
@@ -12,49 +15,34 @@
 #define M_ABOUT_REQUESTED 'abrq'
 
 FortuneWindow::FortuneWindow(void)
-	: BWindow(BRect(0,0,300,300),"Fortune",B_DOCUMENT_WINDOW,B_ASYNCHRONOUS_CONTROLS),
+	: BWindow(BRect(0,0,300,400), "Fortune",
+		B_TITLED_WINDOW, B_NOT_ZOOMABLE | B_ASYNCHRONOUS_CONTROLS),
 	fFortune()
 {
-	BPath path;	
+	SetSizeLimits(300, 20000, 100, 20000);
+
+	BPath path;
 	find_directory(B_SYSTEM_DATA_DIRECTORY, &path);
 	path.Append("fortunes");
 
 	fFortune.SetFolder(path.Path());
 
-	BView *back = new BView(Bounds(),"background",B_FOLLOW_ALL, B_WILL_DRAW);
-	back->SetViewColor(ui_color(B_PANEL_BACKGROUND_COLOR));
-	AddChild(back);
-
-	BButton *close = new BButton(BRect(0,0,1,1),"closebutton","Close",
-								new BMessage(B_QUIT_REQUESTED), B_FOLLOW_RIGHT | B_FOLLOW_BOTTOM);
-	close->ResizeToPreferred();
-	close->MoveTo(Bounds().right - 15 - close->Frame().Width(), 
-				Bounds().bottom - 15 - close->Frame().Height());
-	back->AddChild(close);
-	close->MakeDefault(true);
-	
-	BButton *next = new BButton(BRect(0,0,1,1),"nextbutton","Get Another",
-								new BMessage(M_GET_ANOTHER_FORTUNE), B_FOLLOW_RIGHT | B_FOLLOW_BOTTOM);
-	next->ResizeToPreferred();
-	next->MoveTo(close->Frame().left - 15 - next->Frame().Width(), 
-				Bounds().bottom - 15 - next->Frame().Height());
-	back->AddChild(next);
-	
-	BButton *about = new BButton(BRect(0,0,1,1),"aboutbutton","About" B_UTF8_ELLIPSIS,
-								new BMessage(M_ABOUT_REQUESTED), B_FOLLOW_LEFT | B_FOLLOW_BOTTOM);
-	about->ResizeToPreferred();
-	about->MoveTo(next->Frame().left - 15 - about->Frame().Width(), 
-				Bounds().bottom - 15 - about->Frame().Height());
-	back->AddChild(about);
+	BButton *close = new BButton("closebutton","Close",
+								new BMessage(B_QUIT_REQUESTED));
 	
+	BButton *next = new BButton("nextbutton","Get Another",
```

Comment:
Let's sentence case, "Get another".
It's be useful to make this button the default. Then ou can quickl read a few by hitting ENTER.